docs/index.md

```diff
@@ -5428,6 +5430,67 @@ return api.sign(txJSON, secret); // or: api.sign(txJSON, keypair);
 }
 ```
 
+### Example (multisigning)
+
+```javascript
+const RippleAPI = require('ripple-lib').RippleAPI;
+const api = new RippleAPI({
+  server: 'wss://s.altnet.rippletest.net:51233' 
+
+});
+
+// jon's address has a multi-signing setup with a qourum of 2
+const jon = {
+  account: 'ranYTqDRUKKAqnXAAVDwWBUvvc5op4pXoP',
+  secret: 'shfC5QySdfhCUaEfEF9gxPhAhy58u'
+};
+
+const aya = {
+  account: 'rnrPdBjs98fFFfmRpL6hM7exT788SWQPFN',
+  secret: 'snaMuMrXeVc2Vd4NYvHofeGNjgYoe'
+};
+
+const bran = {
+  account: 'rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH',
+  secret: 'shQtQ8Um5MS218yvEU3Ehy1eZQKqH'
+};
+
+// a transaction which requires multi signing
+const multiSignPaymentTransaction = {
+    TransactionType: 'Payment',
+    Account: 'ranYTqDRUKKAqnXAAVDwWBUvvc5op4pXoP',
+    Sequence: 4,
+    Destination: 'rJ93RLnT1t5A8fCr7HTScw7WtfKJMRXodH',
+    Fee: '120',
+    Amount: '1888000000'
+};
+
+api.connect().then(() => {
+  let tx = JSON.stringify(multiSignPaymentTransaction, null, '\t');
+  
+  // Jon signs the original payment transaction
+  let jonSign = api.sign(tx, jon.secret).signedTransaction;
```

Comment:
If `aya` and `bran` are on the signing list with weight `1` and the quorum is `2`, then Jon should NOT need to sign the transaction. If Jon's master key is not disabled and Jon's secret is known, then there is no security benefit to using multisign in this case.

Reply:
I see. I removed the Jon signing line.

@intelliot All changes are applied.
In this example I just wanted to show the multi-signing with minimum required code. Should I increase the quorum to 3 and add another signer to the list or do you think the current change is enough.